jerry-core/jerry.c

```diff
@@ -2115,6 +2148,11 @@ jerry_value_to_boolean (const jerry_value_t value) /**< input value */
 {
   jerry_assert_api_available ();
 
+  if (ECMA_IS_VALUE_ERROR (value))
+  {
+    return false;
+  }
+
```

Comment:
IMHO, this should be ToBoolean function of the standard: http://www.ecma-international.org/ecma-262/5.1/index.html#sec-9.2
I think the name is good as is.